products/accredited-representative-facing/engineering/unified-sign-in-onboarding.md

```diff
@@ -0,0 +1,40 @@
+# Onboarding
+
+## 🏄 New application using the Unified Sign in Page questionnaire
+
+1. **What is the name of the application?**
+
+   > Accredited Representative Portal (ARP)
+
+2. **What URL endpoints should redirect to your service?**
+
+   > Localhost: localhost:3000/representative/
+   > Dev: dev.va.gov/representative/
+   > Staging: staging.va.gov/representative/
+   > Production: www.va.gov/representative/
+   > 2a. Requires additional query parameters: Not at the moment, but this may be necessary
+   > 2b. Are the URL endpoints dynamic: Not at the moment, but this may be necessary
+
+3. **Which authentication broker are you planning on using?**
+
+   > Sign-in Service (OAuth 2.0)
+
+4. **What type of OAuth integration?** (only applies to OAuth implementation)
+
+   > PKCE
+
+5. **Which service providers are allowed to Sign up?**
+
+   > Login.gov
+
+6. **Which service providers can be used to Sign in?**
+
+   > Login.gov
+
+7. **Does your service have user type requirements?**
```

Comment:
We'll need to figure out how our app presents authn and authz concepts relative to these things like identity verification and LOA.